src/models/converters/transactions/tests/summary.rs

```diff
@@ -203,4 +203,168 @@ fn creation_transaction_to_summary() {
+            "value": "0x65F8236309e5A99Ff0d129d04E486EBCE20DC7B0"
+          },
+          {
+            "name": "value",
+            "type": "uint256",
+            "value": "50000000000000"
+          }
+        ]
+      },
+      "confirmationsRequired": 3,
+      "confirmations": [
+        {
+          "owner": "0x65F8236309e5A99Ff0d129d04E486EBCE20DC7B0",
+          "submissionDate": "2020-08-07T08:51:35.213346Z",
+          "transactionHash": null,
+          "confirmationType": "CONFIRMATION",
+          "signature": "0x2e8da6ff14c68a91caecbfdf71cdd134673f33903404bbaa2624808d7f728d0f551d26342f6eb8afa44cab779921b559311b03b7818088cbe8ca2861c7f473f81b",
+          "signatureType": "EOA"
+        },
+        {
+          "owner": "0xF2CeA96575d6b10f51d9aF3b10e3e4E5738aa6bd",
+          "submissionDate": "2020-08-07T09:04:34.526493Z",
+          "transactionHash": null,
+          "confirmationType": "CONFIRMATION",
+          "signature": "0x1018509bc2fc5f759430fa5740eeb49fe3699932ecf6539f56d42990a4afc782767961c9c88b8f7b5bba6b7b1637128b4b32c82f481f944dc7880787ce2d43b71b",
+          "signatureType": "EOA"
+        },
+        {
+          "owner": "0xBEA2F9227230976d2813a2f8b922c22bE1DE1B23",
+          "submissionDate": "2020-08-07T09:05:06.133804Z",
+          "transactionHash": null,
+          "confirmationType": "CONFIRMATION",
+          "signature": "0x000000000000000000000000bea2f9227230976d2813a2f8b922c22be1de1b23000000000000000000000000000000000000000000000000000000000000000001",
+          "signatureType": "APPROVED_HASH"
+        }
+      ],
+      "signatures": "0x2e8da6ff14c68a91caecbfdf71cdd134673f33903404bbaa2624808d7f728d0f551d26342f6eb8afa44cab779921b559311b03b7818088cbe8ca2861c7f473f81b000000000000000000000000bea2f9227230976d2813a2f8b922c22be1de1b230000000000000000000000000000000000000000000000000000000000000000011018509bc2fc5f759430fa5740eeb49fe3699932ecf6539f56d42990a4afc782767961c9c88b8f7b5bba6b7b1637128b4b32c82f481f944dc7880787ce2d43b71b",
+      "transfers": [
+        {
+          "type": "ERC20_TRANSFER",
+          "executionDate": "2020-08-07T09:06:14Z",
+          "blockNumber": 6975704,
+          "transactionHash": "0x08cf4bb6fe2a7e77e86e7679ec3c266516155cdc9900cdd47afef791169d6e21",
+          "to": "0x65F8236309e5A99Ff0d129d04E486EBCE20DC7B0",
+          "value": "50000000000000",
+          "tokenId": null,
+          "tokenAddress": "0xD9BA894E0097f8cC2BBc9D24D308b98e36dc6D02",
+          "tokenInfo": {
+            "type": "ERC20",
+            "address": "0xD9BA894E0097f8cC2BBc9D24D308b98e36dc6D02",
+            "name": "Compound USDT",
+            "symbol": "USDT",
+            "decimals": 18,
+            "logoUri": "https://gnosis-safe-token-logos.s3.amazonaws.com/0xD9BA894E0097f8cC2BBc9D24D308b98e36dc6D02.png"
+          },
+          "from": "0x1230B3d59858296A31053C1b8562Ecf89A2f888b"
+        }
+      ],
+      "txType": "MULTISIG_TRANSACTION"
+    }"#;
+    let token_info = TokenInfo {
```

Comment:
This could be json too, right?

Reply:
It could. I will replace and extract to a json dumpster module or something. so we can re-use it too.